albatross/cereal/eigen.h

```diff
@@ -20,66 +20,82 @@
 
 namespace cereal {
 
-template <class Archive>
-inline void save(Archive &archive, const Eigen::VectorXd &v) {
-  size_type s = static_cast<size_type>(v.size());
-  archive(cereal::make_size_tag(s));
-  for (size_type i = 0; i < s; i++) {
-    archive(v[i]);
+/*
+ * The subsequent save/load methods catch the serialization methods
+ * for arbitrary Eigen::Matrix* types.  The general idea is that each
+ * row is saved as a Eigen::Vector.
```

Comment:
How was it done before?

Reply:
It was the same idea, but with explict use of `VectorXd`.  Now it's via template specialization with `_Cols=1`.